tokio/src/sync/mpsc/bounded.rs

```diff
@@ -22,6 +22,50 @@ pub struct Sender<T> {
     chan: chan::Tx<T, Semaphore>,
 }
 
+/// A Sender that does not influence RAII semantics, i.e. if all [`Sender`]
+/// instances of a channel were dropped and only `WeakSender` instances remain,
+/// the channel is closed.
```

Comment:
The first line of the documentation for the struct is visible on the module documentation too. It should be a short sentence that fits on a single line on the module documentation.
```suggestion
/// A sender that does not prevent the channel from being closed.
///
/// If all [`Sender`]
/// instances of a channel were dropped and only `WeakSender` instances remain,
/// the channel is closed.
```
+ reflow text